package.json

```
@@ -8,6 +8,7 @@
     "prepare": "husky install",
     "publish": "lerna publish from-package",
     "setup:project": "npx pnpm@7.3.0 i && node scripts/setup-env-files.js && npx pnpm build",
+    "dispose:nm": "node scripts/dispose-node-modules.js",
```

```suggestion
    "clean": "lerna clean",
```

a small suggestion, indeed of manually doing it, we can use `lerna clean` command for this :)